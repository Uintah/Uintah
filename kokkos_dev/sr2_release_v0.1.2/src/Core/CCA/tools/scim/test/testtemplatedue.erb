
inPackage=<%= $inPackage%>
outPackage=<%= $outPackage%>

<port> 
  PORT
  <%= $inPortName%>
  <%= $outPortName%>
  EOF_PORT
</port>

<port>
 <method> 
  PORTMETHOD
  <%= $inMethodName%>
  <%= $outMethodName%>
  EOF_PORTMETHOD
 </method>
</port>
