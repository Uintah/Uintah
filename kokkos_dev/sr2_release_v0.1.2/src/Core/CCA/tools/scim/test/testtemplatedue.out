
inPackage=
outPackage=

   
  PORT
  iStringPort
  oStringPort
  EOF_PORT
   
  PORT
  Margaritaville
  Margaritaville
  EOF_PORT


  
        
  PORTMETHOD
  getString
  getString
  EOF_PORTMETHOD
    
  PORTMETHOD
  m1
  m2
  EOF_PORTMETHOD
 
  
        
  PORTMETHOD
  jimmy_buffet
  jimmy_buffet
  EOF_PORTMETHOD
 

